runtime/runtime/src/lib.rs

```diff
@@ -2489,6 +2544,81 @@ mod tests {
             .expect("Compiled contract should be cached")
             .expect("Compilation result should be non-empty");
     }
+
+    #[cfg(feature = "protocol_feature_compute_costs")]
+    #[test]
+    fn test_compute_usage_limit() {
+        let initial_balance = to_yocto(1_000_000);
+        let initial_locked = to_yocto(500_000);
+        let (runtime, tries, root, mut apply_state, signer, epoch_info_provider) =
+            setup_runtime(initial_balance, initial_locked, 1);
+
+        let mut free_config = RuntimeConfig::free();
+        let sha256_cost =
+            ParameterCost { gas: Gas::from(1u64), compute: Compute::from(10_000_000_000_000u64) };
+        free_config.wasm_config.ext_costs.costs[ExtCosts::sha256_base] = sha256_cost.clone();
+        apply_state.config = Arc::new(free_config);
+        // This allows us to execute 1 receipt with a function call per apply.
+        apply_state.gas_limit = Some(sha256_cost.compute);
+
+        let mut receipts = create_receipts_with_actions(
+            alice_account(),
+            signer.clone(),
+            vec![Action::DeployContract(DeployContractAction {
+                code: near_test_contracts::rs_contract().to_vec(),
+            })],
+        );
+
+        for _ in 0..2 {
```

Comment:
please add comment: add 2 receipts but only expect one to be executed due to compute limit

Reply:
Done.